config/production.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Delete this and the QA config -- we can put the environment variables directly in the lambda when we set it up